### PR TITLE
Add Collaborators Attribution

### PR DESCRIPTION
**Pull request by curran**
image

image

### CODE REVIEW

**Comment by curran** [comment]
@seemantk Ready for review. Any thoughts on this one?

**Comment by seemantk** [comment]
Hi Curran, let's chat about this one, as I'm not part of Datavis.tech, so the attribution will need adjustment.

**Comment by curran** [comment]
@seemantk Possibly we can add our names there? Feel free to make proposed changes here on this PR.

**Comment by seemantk** [comment]
@curran thanks, I think our names will be good as attribution. Have not had too much spare cycles to think on this one, so will address at weekend, if that's ok?

**Comment by curran** [comment]
Sure no problem.

Some options:

Option A
image

Option B
image

Option C
image

Would be curious to hear if any of these would suit you. We can make our names links perhaps as well.

**Comment by curran** [comment]
Option D
image

**Comment by seemantk** [comment]
Hi Curran,

thank you.  none of the changes you've proposed handles the issue that I brought up, which is that Datavis.tech's attribution is the problematic one:

1. Datavis.tech is you and you alone
2. So, putting your name and your company is twice the attribution

So, out of the two (your name or your company), please feel free to pick one or the other.

Cheers,
Seemant

**Comment by curran** [comment]
Sure, so let's go with option D, with just our names then. That sounds good to me.

File: _layouts/default.html [hunk 1]
```diff
@@ -98,9 +98,12 @@
     </section><!--.col-md-12-->
   </div><!--.row-->
 
-  <div class="row datavis-tech-attribution">
+  <div class="row collaborators">
     <div class="col-md-12 text-center">
-      <span class="text-muted">Crafted with <i class="fa fa-heart"></i> by Seemant Kulleen and Curran Kelleher</span>
+      <span class="text-muted small">
+        <p>Data Vis. Tech. by <a href="http://seemantk.com">Seemant Kulleen</a> and <a href="http://datavis.tech">Curran Kelleher</a>.
```
Comment (curran) [inline _layouts/default.html]: Nice!

**Comment by curran** [comment]
One additional thought - if folks land on the visualization page, there's no link back to the GitHub project. Maybe that link could go in this area as well?

**Comment by curran** [comment]
Closing this, as the footer will be CFI's content.

**Pull request closed without merging**